functions/telemetry.js

```diff
@@ -0,0 +1,41 @@
+const fetch = require('node-fetch')
+
+const TELEMETRY_SERVICE_URL = 'https://cli-telemetry.netlify.engineering'
+
+// This function is a workaround for our inability to redirect traffic to a ntl function in another site
+// using redirects (see https://github.com/netlify/cli-telemetry-service/issues/14)
+const handler = function ({ path, httpMethod, headers, body }) {
```

Comment:
I think we need to explicitly use `async` in the signature otherwise we need to invoke the `callback` argument.

Actually we might need to use `callback` with `callbackWaitsForEmptyEventLoop = false` to get the outcome we want. See https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html.

With `callbackWaitsForEmptyEventLoop = false` there's the side effect of freezing the event loop

Reply:
> With callbackWaitsForEmptyEventLoop = false there's the side effect of freezing the event loop

yeah I was just reading through it and I don't think that will cut it also 😕 because we would ideally want the processing to continue but not be frozen until the next execution.

I think we might need to just wait for the response, at least for now.

Reply:
Yeah, tried it out under the latest commit, managing connection state becomes an issue:

![image](https://user-images.githubusercontent.com/5799039/119388670-81113d00-bcc2-11eb-87b7-9433283aaf2b.png)

Logs:
https://app.netlify.com/sites/cli/functions/telemetry?scope=deploypreview:2432
https://app.netlify.com/sites/cli-telemetry-proxy/functions/track

Reply:
Changed the logic to wait for the response 👍

Reply:
FYI, in terms of response time, getting between 600ms to 1s (for cold executions)